jwst/resample/resample.py

```diff
@@ -377,6 +377,14 @@ def drizzle_arrays(insci, inwht, input_wcs, output_wcs, outsci, outwht, outcon,
         if outcon.ndim == 3:
             outcon = outcon[planeid]
 
+        if xmin is xmax is ymin is ymax is None:
+            bb = input_wcs.bounding_box
+            ((x1, x2), (y1, y2)) = bb
```

Comment:
very minor thing: IMO no need to split this into two lines

Reply:
Thanks.  Copy-and-pasted from somewhere else where `bb` was used subsequently 200 lines down.  😂